prov/usnic/src/usdf_cm.c

```diff
@@ -235,13 +238,32 @@ usdf_cm_msg_connreq_failed(struct usdf_connreq *crp, int error)
 		eq = pep->pep_eq;
 	}
 
+	reqp = (struct usdf_connreq_msg *) crp->cr_data;
```

Comment:
I don't think I care if you separate this function into reject/non-reject versions, you have a better feel for what that refactor looks like.

I do think there's probably some opportunity for cleanliness/refactoring in the CM code, but my preference is to get this version of the code in and then do our refactoring later.  Right now we don't even properly support reject, so this PR is a big functionality improvement .